syntax = "proto2";
import "circuit.proto";
package psjjjj;

message RequestProto {
    required int32 type = 1; // 0: VHDL parsing; 1: Simulation; 2: Get Module List
    optional string vhdl_code = 2;
    optional CircuitProto circuit = 3;
}

